code/modules/mining/abandoned_crates.dm

```diff
@@ -188,7 +188,9 @@
 			new /obj/item/clothing/suit/costume/wellworn_shirt/graphic/ian(src)
 			new /obj/item/clothing/suit/hooded/ian_costume(src)
 		if(67 to 68)
-			new /obj/item/toy/plush/awakenedplushie(src)
+			var/obj/item/gibtonite/free_bomb = new /obj/item/gibtonite(src)
```

Comment:
This is not just the gibtonite exploding. It also causes the crate to explode. Is it actually a good idea to punish people for successfully opening the crate?

Reply:
There are overwhelmingly more good outcomes than bad still, and some might not even see this as a bad outcome since the gibtonite can be disarmed.